### PR TITLE
allow transform function to return array of docs

### PR DESCRIPTION
**Pull request by gr2m**
hey @glynnbird let me know if you want to add that functionality to `couchimport` (or if I missed something). We use the script to import a big CSV table but need to split up the rows in multiple docs, so we need our transform script to be able to return an array. That PR allows for that, but it has no tests or documentation yet, waiting for your feedback first

### CODE REVIEW

**Comment by glynnbird** [comment]
This is an awesome idea!

I'd be happy to accept this with example usage in the README (and a test or two if you have time).

Thanks for the contribution @gr2m :)

**Comment by gr2m** [comment]
I've updated the README and added an example. I’m not sure how / where to add the test for the transform though. How would you approach it?

File: examples/transform_split_docs.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    _id: row.id,
+    name: row.name,
+    type: 'person'
+  }
+  var doc2 = {
+    _id: row.company_id,
+    name: row.company_name,
+    type: 'company'
+  }
+  for(var i in doc) {
```
Comment (glynnbird) [inline examples/transform_split_docs.js]: I'm not sure what this example is doing. It seems to be iterating over a variable "doc" which doesn't appear anywhere else? :)

**Comment by glynnbird** [comment]
@gr2m Don't worry about tests for now. I'm going to write some 'transform' tests shortly and I'll fold your changes into that work :)

File: examples/transform_split_docs.js (hunk 1 above)
Comment (gr2m) [reply examples/transform_split_docs.js]: oops sorry that loop makes no sense I’ve no idea what happened :)

Comment (gr2m) [reply examples/transform_split_docs.js]: fixed in 8d6eda9

**Pull request merged**